src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/rca/OldGenRca.java

```diff
@@ -100,6 +104,7 @@ protected int getFullGcEventsOrDefault(final double defaultValue) {
 
   protected double getOldGenUsedOrDefault(final double defaultValue) {
     if (heap_Used == null) {
+      StatsCollector.instance().logException(StatExceptionCode.MISCONFIGURED_OLD_GEN_RCA);
```

Comment:
Name this one something different so we can tell it apart from line 81

Reply:
I thought one sparse metric was enough. Added them still, just to avoid confusion.